examples/converting-to-user-defined-type/converting_to_user_defined_type.metatags

```diff
@@ -1,2 +1,2 @@
 description: This BBE demonstrates converting to a user-defined type using `cloneWithType`
```

Comment:
```suggestion
description: This BBE demonstrates converting to a user-defined type using `cloneWithType`.
```
Converting what?

Reply:
Changed to `This BBE demonstrates converting JSON to a user-defined type using cloneWithType.`